corehq/apps/app_manager/tests/test_app_manager.py

```diff
@@ -267,7 +267,7 @@ def testPruneAutoGeneratedBuilds(self, mock):
 
         # Build #2, auto-generated
         app.save()
-        make_build(app)
+        autogenerate_build(app, "username")
```

Comment:
How are these two the same?
`make_build` was generating build instantly where as `autogenerate_build` would queue a task?

Reply:
Celery runs synchronously when called inside tests, I believe.  It passes locally without celery running.  I did consider calling the task directly, without the `.delay`, but thought this would be preferable, as it more closely tests the code run in the view code.

Reply:
Can you confirm by running the test and see that that code is actually called? I don't think or will be surprised if we set `CELERY_EAGER` or something similar for all tests, otherwise it would start doing everything(emails etc) and take longer to run.

Reply:
I just confirmed that.  The asserts below also validate that a build was created and then deleted.